src/build.props

```diff
@@ -53,8 +53,8 @@
     <Choose>
       <When Condition="$(OS) == 'WINDOWS_NT'">
         <PropertyGroup>
-          <LibTargetFrameworks>net48;net6.0</LibTargetFrameworks>
-          <TestTargetFrameworks>net48;net6.0</TestTargetFrameworks>
+          <LibTargetFrameworks>netstandard2.0</LibTargetFrameworks> <!--netstandard 2.0 is the highest version that still supports net48 -->
+          <TestTargetFrameworks>net48;net6.0</TestTargetFrameworks> <!-- for testing we just want to check latest on framework and core -->
```

Comment:
I think this is a bad reversion. The targets were changed to be explicitly `net48;net6.0` on Windows in the last PR that added the build checks, and the comments were removed since they're no longer relevant.